docs/pages/technical/fhir-rules.rst

```diff
@@ -136,8 +145,8 @@ Organization
 Source
 ......
 
-The :code:`source` will always be an :code:`sourceAttachment`. The attachment must have a :code:`contentType` and must have an :code:`url`.
-There are several valid contentTypes:
+The :code:`source` will always be an :code:`sourceAttachment`. The source always points to the latest change in consent proof.
+The attachment can have a :code:`contentType` and must have an :code:`url`. There are several valid contentTypes:
```

Comment:
How can the URL be mandatory in the use-case that the proof is some document in a archive?

Reply:
in the case if consent has been pulled due to legal reasons, a document is probably available. We could (for now) also allow plain-text proof for updates.
The original proof should still be available and queryable, if really archived we could suggest to let the API's return an explanation. Or in general a 404 for older versions means archived and "please contact the care organization directly"